app/views/content/dates_and_deadlines.md

```diff
@@ -1,9 +1,22 @@
+### <%= CycleTimetable.cycle_year_range(RecruitmentCycle.current_year - 1) %> recruitment cycle – current
```

Comment:
@tomas-stefano is there a better way to display the current recruitment cycle ("2021 to 2022")? `CycleTimetable.cycle_year_range` seems to return the academic cycle not the recruitment cycle?

Reply:
`RecruitmentCycle.cycle_name(RecruitmentCycle.current_year)` ?

Reply:
What Jon said, thank you Jon!